x-pack/plugins/observability/public/components/slo/error_rate_chart/use_lens_definition.ts

```diff
@@ -555,7 +556,7 @@ export function useLensDefinition(slo: SLOResponse): TypedLensByValueInput['attr
       adHocDataViews: {
         '32ca1ad4-81c0-4daf-b9d1-07118044bdc5': {
           id: '32ca1ad4-81c0-4daf-b9d1-07118044bdc5',
-          title: '.slo-observability.sli-v2.*',
+          title: SLO_DESTINATION_INDEX_PATTERN,
```

Comment:
This resolves to `.slo-observability.sli-v2*`, not `.slo-observability.sli-v2.*`, fyi.

Reply:
`.slo-observability.sli-v2.*` matches the indices where the data is actually stored, e.g. `.slo-observability.sli-v2.YYYY-MM-DD` without the empty index `.slo-observability.sli-v2`.

While `.slo-observability.sli-v2*` matches the monthly indices + the empty index (created to please the transform).

Both options are fine. Using the destination index pattern constant is the way to go